activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/services/audit/ErrorAuditProducerIT.java

```diff
@@ -0,0 +1,124 @@
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
+public class ErrorAuditProducerIT {
+
+    private static final String ERROR_START_EVENT_SUBPROCESS = "errorStartEventSubProcess";
+    
+    @Autowired
+    private AuditConsumerStreamHandler streamHandler;
+    
+    @Autowired
+    private ProcessInstanceRestTemplate processInstanceRestTemplate;
+
+    @Before
+    public void setUp() {
+        streamHandler.clear();
+    }
+
+    @Test
+    public void should_produceBpmnErrorEvents_when_processIsExecuted() {
+        
+        ResponseEntity<CloudProcessInstance> startProcessEntity = processInstanceRestTemplate.startProcess(new StartProcessPayloadBuilder()
+                                                                                                            .withProcessDefinitionKey(ERROR_START_EVENT_SUBPROCESS)
+                                                                                                            .withName("processInstanceName")
+                                                                                                            .withBusinessKey("businessKey")
+                                                                                                            .build());
+
+        assertThat(startProcessEntity.getStatusCode()).isEqualTo(HttpStatus.OK);
+        
+        CloudProcessInstance processInstance = startProcessEntity.getBody();
+
+        await().untilAsserted(() -> {
+            assertThat(streamHandler.getReceivedHeaders()).containsKeys(RUNTIME_BUNDLE_INFO_HEADERS);
+            assertThat(streamHandler.getReceivedHeaders()).containsKeys(ALL_REQUIRED_HEADERS);
+            List<CloudRuntimeEvent<?, ?>> receivedEvents = streamHandler.getAllReceivedEvents();
+
+            assertThat(receivedEvents)
+                    .filteredOn(CloudBPMNErrorReceivedEvent.class::isInstance)
+                    .extracting(CloudRuntimeEvent::getEventType,
+                                CloudRuntimeEvent::getProcessDefinitionId,
+                                CloudRuntimeEvent::getProcessInstanceId,
+                                CloudRuntimeEvent::getProcessDefinitionKey,
+                                CloudRuntimeEvent::getProcessDefinitionVersion,
+                                CloudRuntimeEvent::getBusinessKey,
+                                event -> bpmnError(event).getElementId(),
+                                event -> bpmnError(event).getProcessDefinitionId(),
+                                event -> bpmnError(event).getProcessInstanceId(),
+                                event -> bpmnError(event).getErrorCode(),
+                                event -> bpmnError(event).getErrorId(),
+                                event -> bpmnError(event).getActivityType(),
+                                event -> bpmnError(event).getActivityName()
+                    )
+                    .containsExactly(
+                            tuple(ERROR_RECEIVED,
```

Comment:
Could you check the events `ACTIVITY_STARTED` and `ACTIVITI_COMPLETED` for both throw and catch error, as well?

Reply:
I have extended a test